src/main/resources/static/schema.graphql

```diff
@@ -54,4 +56,26 @@ enum RegionType {
     LOCATION_6
     LOCATION_7
     LOCATION_8
+}
+
+#item
+type Item {
+    id: ID
+    name: String
+    description: String
+    inStock: Int
+    image: String
+    region: RegionType
+    lastAccessedDate: DateTime
+    categoryId: Int
+}
+
+type ItemPageInfo {
+    totalElements: Int
+    totalPages: Int
+    size: Int
+    number: Int
+    numberOfElements: Int
+    sort: String
```

Comment:
"sort" should be array of strings. Take a look at "sort" in "getCategories" as example.